### PR TITLE
Add examples in all formats for each property

### LINKED ISSUE #18: EPUB example needed accessibilityHazard
Should also include EPUB example
```
<package …>
   <metadata>
      …
      <meta
          property="schema:accessibilityHazard">
         flashingHazard
      </meta>
      <meta
          property="schema:accessibilityHazard">
         noMotionSimulationHazard
      </meta>
      <meta
          property="schema:accessibilityHazard">
         noSoundHazard
      </meta>
     	…
   </metadata>
   …
</package>
```

Should we also include the "none" case which would be the most common?

### LINKED ISSUE #41: Add previous version link
Trying to find out when the tableOfContents value got misplaced made me wish it was possible to view old versions of the vocabulary. I had to go to the wayback machine before discovering it was missing from the wiki.

Propose we manually add a "previous version" link and keep a dated copy of each release. It only adds one simple extra step to the release process (i.e., to copy the dated file that respec generates and then replace the index.html file in /latest).

### PR DESCRIPTION
**Pull request by mattgarrish**
This pull request adds examples in json-ld, epub, rdfa, and microdata for each of the properties.

It also introduces a method of switching between examples so that only one format is displayed at any time.

Fixes #18
Fixes #41

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/a11y-discov-vocab/pull/36.html" title="Last updated on Feb 8, 2022, 3:35 PM UTC (37342b9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/a11y-discov-vocab/36/a5ad75f...37342b9.html" title="Last updated on Feb 8, 2022, 3:35 PM UTC (37342b9)">Diff</a>

### CODE REVIEW

**Review by clapierre — APPROVED** [review]
I think this is good hard to read without seeing it live, may have additional comments once I see it merged in and I can switch between the different example types.

**Comment by mattgarrish** [comment]
@clapierre if you want to try it out, use the preview link.

The diff version won't work because it's generated by a W3C tool so the scripts will be missing.

**Comment by madeleinerothberg** [comment]
Do you want feedback on every example, or just approval on the functionality (which is awesome)? Here's a nit pick:

`AccessibilityControl is not applicable to EPUB`

It isn't applicable to most EPUBs, but if there are interactive components (STEM simulations in a textbook, etc.) it would be relevant.

**Comment by clapierre** [comment]
Now that I see the Diff I have a few suggestions.
First off in some the EPUB examples the metadata includes
```
      <meta
          property="rdf:type">
         https://schema.org/Book
      </meta>
```
Should this be included in every example?
I assume this would be a good suggestion for all publishers to include in their metadata right?

I think for the examples for accessibilityHazard since we are showing "flashing" we should then have "noMotionSimulationHazard" and "noSoundHazard" to be complete.

As for the accessibilityFeatures example, I would rather see "printPageNumbers" used instead of "transcript" as that would be more often found within an EPUB.

And finally I would think that at the top level for each example we should say what the example is showing.
IE as for the case with accessibilityHazards
"Content which contains a video at a dance club with a strobe light flashing in the background"

for accessMode
"Content which contains both content images as well as text"

Similar for the other examples.

**Comment by mattgarrish** [comment]
> Do you want feedback on every example

If there's anything you spot, yes, please report it.

> It isn't applicable to most EPUBs, but if there are interactive components (STEM simulations in a textbook, etc.) it would be relevant.

Yes, that's probably too harsh a statement. The problem we've run into is that it's not clear how you can make a claim of being able to control the content when it also depends on the reading system/operating system and AT you're using. It becomes a statement that in ideal conditions, it could be controlled, but practically the author really doesn't know because there are so many combinations that would have to be tested.

**Comment by mattgarrish** [comment]
> I assume this would be a good suggestion for all publishers to include in their metadata right?

It's a bit of metadata overkill, to be honest. Does it matter if you assume CreativeWork or explicitly declare Book? I can't make a case for why.

I only used it in the spirit of showing some variation (i.e., how you can handle alternative schema.org types in EPUB if you want to).

**Comment by mattgarrish** [comment]
> And finally I would think that at the top level for each example we should say what the example is showing.

I'll have to do some tinkering to get that work. Right now, adding a description collides with the dynamic inserting of the buttons to change the format.

**Review by clapierre — APPROVED** [review]
Looks great Matt, love the new and improved examples!

**Pull request merged**
